src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -193,7 +193,14 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 						currentAudioTrack.setLang(getLang(MI.Get(audio, i, "Language/String")));
 						currentAudioTrack.setAudioTrackTitleFromMetadata((MI.Get(audio, i, "Title")).trim());
 						currentAudioTrack.getAudioProperties().setNumberOfChannels(MI.Get(audio, i, "Channel(s)"));
-						currentAudioTrack.setSampleFrequency(getSampleFrequency(MI.Get(audio, i, "SamplingRate")));
+						value = MI.Get(audio, i, "SamplingRate");
+						if (isNotBlank(value)) {
+							if (!value.contains("/")) {
+								currentAudioTrack.setSampleFrequency(getSampleFrequency(MI.Get(audio, i, "SamplingRate")));
```

Comment:
Because i've a much more limited global view/understanding of the code, not like you...

I can try to see what i can do.